File: answer_checker.R

for(i in candidates) {
  answer <- scan(file=paste(c("data/정답텍스트/"), i, c(".txt"), sep=""), what="char", sep="\n" ,quote=NULL)
  myanswer <- scan(file=paste(c("answer/"), i, c(".txt"), sep=""), what="char", sep="\n" ,quote=NULL)
  print(i)
  print(which( answer != myanswer ))
}

File: answer_checker.py
import filecmp
candidates = ["문재인","사회자","심상정","안철수","유승민","홍준표"]

for i in candidates:
    print(i)
    print(filecmp.cmp("answer/" + i + ".txt", "data/정답텍스트/" + i + ".txt"))
